libs/server/Resp/RespServerSession.cs

```diff
@@ -54,9 +54,10 @@ internal sealed unsafe partial class RespServerSession : ServerSessionBase
         internal readonly TransactionManager txnManager;
         readonly ScratchBufferManager scratchBufferManager;
 
+        SessionParseState parseState;
         GCHandle recvHandle;
         byte* recvBufferPtr;
-        int readHead;
+        int readHead, endReadHead;
         byte* dcurr, dend;
         bool toDispose;
```

Comment:
Need comment describing difference between readHead and endReadHead. And add some for the other pointers while you're in here.